src/utils/data_loaders.py

```diff
@@ -173,9 +174,12 @@ def process_sentence(tokenizer_name, sent, max_seq_len, sos_tok=SOS_TOK, eos_tok
     '''process a sentence '''
     max_seq_len -= 2
     assert max_seq_len > 0, "Max sequence length should be at least 2!"
-    TOKENIZER = AVAILABLE_TOKENIZERS[tokenizer_name]
+    tokenizer = get_tokenizer(tokenizer_name)
+    if tokenizer_name.startswith("bert"):
```

Comment:
This seems a little odd, because it overrides the `sos_tok` and `eos_tok` that are passed in as parameters. Are these parameters actually used anywhere, or do we always take the defaults? 

If the latter, we should just remove them and have:
```
if tokenizer_name.startswith("bert-"):
    sos_tok, eos_tok = BERT_SEP_TOK, BERT_CLS_TOK
else:
    sos_tok, eos_tok = SOS_TOK, EOS_TOK

Reply:
Yeah we're taking the defaults almost always